gateway/1.5/modules/ROOT/pages/policies-pdk-debug-local.adoc

```diff
@@ -1,5 +1,145 @@
+
+The `run` command only supports Flex Gateways running in Local Mode in a Docker Container. To register a Flex Gateway, see the relevant tutorial for your preferred credentail type:
+
+* xref:local-reg-run-up.adoc#docker[]
+* xref:local-reg-run-app.adoc#docker[]
+* xref:local-reg-run-token.adoc#docker[]
+
+IMPORTANT: You only need to to run the registration command. Do not run the Docker start command. The `run` command completes this step.
+
+A different registration file must be created on each device testing the policy.
+
+The registration file is ignored for storage in remote repositories and is listed in .gitignore file located at the root of the project directory.
+
+
+== Configure the Policy
+
+Depending on the configurations listed in the `gcl.yaml` file you configured in xref:policies-pdk-create-schema-definition.adoc[], you may have to configure some parameters before running the policy locally.
+
+For example, if the `definition/gcl.yaml` file is as follows:
+
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com.v1alpha1
+kind: Extension
+metadata:
+  labels:
+    title: my-custom-policy
+    category: Security 
+spec:
+  extends:
+    - name: extension-definition
+  properties: 
+    user:
+      type: string
+      default: user1
+    password:
+      type: string
+  required:
+    - someConfiguration
+
+----
+
+Update the `/test/config/api.yaml` file with your configuration value:
+
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: ApiInstance
+metadata:
+  name: ingress-http
+spec:
+  address: http://0.0.0.0:8081
+  services:
+    upstream:
+      address: http://backend
+      routes:
+        - config:
+            destinationPath: /anything/echo/
+  policies:
+    - policyRef:
+        name: my-custom-policy
+      config:
+        password: 12345678
+
+---- 
+
+In this example, the `user` is set to the default value `user1`, and `password` is set to `12345678`.
+
+== Add Additional Policies
+
+You can also apply additional policies to the test API instance to test your custom policies behavior alongside other policies.
+
+However, your custom policy under development must be the first policy specified, for example:
+
+[source,yaml]
+----
+  policies:
+    - policyRef:
+        name: my-custom-policy
+      config:
+        password: 12345678
+    - policyRef:
+        name: additional-policy
+      config:
+        
+----
+
+== Deploy the Policy with PDK
+
+After you complete the previous steps, run the `run` command from the policy's root directory to deploy policy:
+
+[source,cmd]
+----
+make run
+----
+
+This command start the two Docker containers in the `<root-directory>/test/config/docker-compose.yaml` file:
```

Comment:
I think this path is wrong, my fault, it should be:  `<root-directory>/test/docker-compose.yaml`